chan_handler.go

```diff
@@ -1,10 +1,12 @@
-// Copyright 2013-2016 Apcera Inc. All rights reserved.
+// Copyright 2013-2017 Apcera Inc. All rights reserved.
 
 package graft
 
 // ChanHandler is a convenience handler when a user wants to simply use
 // channels for the async handling of errors and state changes.
 type ChanHandler struct {
+	LogPositionHandler
```

Comment:
Agree, the name might be confusing. I decided against "state" because it was also confusing. There are already a lot of references to "state" in graft with respect to node state changes.